src/adapter/src/coord/sequencer.rs

```diff
@@ -2193,11 +2193,19 @@ impl<S: Append + 'static> Coordinator<S> {
                 .add_transaction_ops(TransactionOps::Peeks(peek_plan.timestamp_context.clone()))?;
         }
 
+        let timestamp = peek_plan.timestamp_context.timestamp().cloned();
```

Comment:
I don't love this clone because it'll almost never be used. But also: probably doesn't matter.

Reply:
Why not move it into the `if` statement below?

Reply:
`peek_plan` gets moved so isn't available anymore.